src/test/java/guitests/SortTest.java

```diff
@@ -50,5 +50,19 @@ public void sortTest () {
         press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
         type("-date");
         push(KeyCode.ENTER);
+        // Issue state (open)
+        doubleClick("#dummy/dummy_col0_filterTextField");
+        doubleClick("#dummy/dummy_col0_filterTextField");
+        type("sort");
+        press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
+        type("status");
```

Comment:
So it's not possible to type "sort:status" in one shot? If cannot, perhaps we should write our own wrapper method to take a string and convert it to the correct key strokes?